subworkflows/local/make_chunks/make_chunks.nf

```diff
@@ -0,0 +1,67 @@
+include { BCFTOOLS_INDEX                    } from '../../../modules/nf-core/bcftools/index/main'
+include { BCFTOOLS_INDEX as BCFTOOLS_INDEX_2} from '../../../modules/nf-core/bcftools/index/main'
+include { BCFTOOLS_INDEX as BCFTOOLS_INDEX_3} from '../../../modules/nf-core/bcftools/index/main'
+include { GLIMPSE_CHUNK                     } from '../../../modules/nf-core/glimpse/chunk/main'
+include { BCFTOOLS_CONVERT                  } from '../../../modules/nf-core/bcftools/convert/main'
+include { BCFTOOLS_NORM                     } from '../../../modules/nf-core/bcftools/norm/main'
+include { BCFTOOLS_VIEW                     } from '../../../modules/nf-core/bcftools/view/main'
+
+
+workflow MAKE_CHUNKS {
+
+    take:
+    ch_reference                           // channel: [ val(meta),vcf ]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    // Make chunks
+    ch_vcf_csi_chr = ch_reference.map{meta, vcf, csi -> [meta, vcf, csi, meta.chr]}
+    GLIMPSE_CHUNK(ch_vcf_csi_chr)
+
+    // Rearrange chunks into channel
+    ch_chunks = GLIMPSE_CHUNK.out.chunk_chr
+                    .splitText()
+                    .map { metamap, line ->
+                        def fields = line.split("\t")
+                        def startEnd = fields[2].split(':')[1].split('-')
+                        [metamap, metamap.chr, startEnd[0], startEnd[1]]
+                    }
+
+    ch_fasta = Channel.value([file(params.fasta,checkIfExists: true)])
+                            .map { file -> [[id: 'genome'], file] }
+
+    // Join duplicated biallelic sites into multiallelic records
```

Comment:
I agree that we should only have one subworkflow for panel pre-processing. However, there are differences in how we do that. For example, in BCFTOOLS_NORM you use "-m","-any" while I use "-m","+any". The same happens in bcftools_view (-v snps -Oz vs. "-m 2", "-M 2", "-v snps"). The preparation for QUILT is different than for GLIMPSE. If you agree, I'll open an issue and can try to re-arrange this code in GET_PANEL in a second, separate PR.

Reply:
I will address this in a different PR because I need to change the structure of `GET_PANEL`. There are a few steps that are shared among `GLIMPSE` and `QUILT` but some outputs are unique for each. Therefore, in order to avoid calculating things that won't be used (for example, if the user only sets `--tools quilt`, some outputs from the current version that were originally designed for `glimpse` are not needed). I will then separate the steps of this subworkflow to be able to handle that.